### DISCUSSION TITLE
New translation in Chinese Mandarin (Simplified)

### DISCUSSION THREAD

**Discussion by heymenshan** [sircharlo/meeting-media-manager#2480] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Chinese Mandarin (Simplified) (CHS/cmn-hans).

**Comment by mtdvlpr** [comment]
Thank you so much for your willing spirit!
If you would like to volunteer to become a M³ language moderator for a language you speak, please [sign into Crowdin](https://crowdin.com/project/meeting-media-manager) using your GitHub account, and then let me know once that is done.

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Reply by heymenshan** [reply]
I have finished translating all the translation strings. Hope the translations could be applied in the near future.

**Reply by sircharlo** [reply]
Thank you @heymenshan !

**Comment by mtdvlpr** [comment]
Hi @heymenshan, thanks for your work and patience. Your translations have been implemented at #2529 and will be released soon.

**Comment by heymenshan** [comment]
Thank you so much!
